index.html

```diff
@@ -298,9 +299,9 @@ <h1 class="main-text">About<br />Myself</h1>
                   alt="arrow icon"
                 />
               </figure>
-              </ul>
             </li>
-
+            
+            <div class="about-languages-container">
             <li class="dropdown-menu">
               <img src="images/javascript-icon.png" alt="javascript icon" />
```

Comment:
- [ ] Kindly chnage the dropdown button to have a replica of the original figma deisgn. Have in mind that these designs are mostly made by the product team, of which you the developer has to implement. So the design is not yours to change but at the decertion of the client or product manager 

- [ ] The social icons adjacent to the left of on this section of page needs to be completed. yours has only `3`. Kindly note that you'll have to put your real handle in them in future milestones, of which having them complete just as in the design is very necessary 👍  

### Your work:
![image](https://github.com/JavierAybar/Portfolio/assets/50990925/7cfb4e00-bb78-499e-92fd-620845eecb54)

### Figma template:
![image](https://github.com/JavierAybar/Portfolio/assets/50990925/a0d50fe8-cbfb-4f7b-b66e-d5a63f09a986)